examples/main.tf

```diff
@@ -0,0 +1,16 @@
+module "aws_vpc" {
+  source               = "./.."
+  cidr_block           = "172.16.0.0/16"
```

Comment:
Is it possible to have another example with CIDR block from AWS IPAM?

Reply:
Yes, just a moment I add that @JohnNMwangi